xcodeproj/pathobjects.py

```diff
@@ -116,6 +124,9 @@ def absolute_path(self) -> Optional[str]:
         if path is None:
             return None
 
+        if path.startswith("/"):
```

Comment:
This is a little hacky, but the entire thing is based on the idea that every path is relative. Turns out, that's not always the case.